docker/jupyterhub.sh

```diff
@@ -217,9 +217,10 @@ function set-default-configs() {
 function set-default-name-pattern() {
   if [ -z "${CLUSTER_NAME_PATTERN}" ];
   then
-    default_name_pattern="dataprochub-$( curl -s ${metadata_base_url}/instance/name -H "Metadata-Flavor: Google" )-{}"
+    default_name_pattern="dataprochub-{}"
     echo "Default name pattern not specified. Setting default name pattern to ${default_name_pattern}"
     export CLUSTER_NAME_PATTERN="${default_name_pattern}"
+    export ALLOW_RANDOM_CLUSTER_NAMES="true"
```

Comment:
Any advantages for forcing the add of a random string?
With named server, same users can now have multiple clusters and choose the name for each cluster.

Reply:
This was tied to the change above that changed the default cluster name pattern. I've reverted that change, so also reverting this.